### PR TITLE
dreambooth updates

### PR DESCRIPTION
**Pull request by wtomin**
Train dreambooth and inference. I created `LatentDiffusionDreamBooth` instead of using `LatentDiffusionDB` because `LatentDiffusionDB` rewrites many functions.

### CODE REVIEW

**Comment by wtomin** [comment]
@SamitHuang @zhaoyuzhi @DongyaoZhu @zhanghuiyao Feel free to review. I forgot to add reviewers ahead.

**Comment by DongyaoZhu** [comment]
is it possible to add scripts for finding rare token identifiers?

**Review by DongyaoZhu — APPROVED** [review]
lgtm for functionality. Eventually we need to wrap db, controlnet, etc. with a common interface

File: examples/stable_diffusion_v2/ldm/models/diffusion/ddim.py [hunk 1]
```diff
@@ -260,7 +260,7 @@ def p_sample_ddim(
                     c_in.append(ops.concat([unconditional_conditioning[i], c[i]], axis=0))
             else:
                 c_in = ops.concat([unconditional_conditioning, c], axis=0)
-            model_uncond, model_t = ops.split((self.model.apply_model(x_in, t_in, c_crossattn=c_in)), 0, 2)
+            model_uncond, model_t = ops.Split(0, 2)(self.model.apply_model(x_in, t_in, c_crossattn=c_in))
```
[the remark this answers is not in the document]
Comment (wtomin) [reply examples/stable_diffusion_v2/ldm/models/diffusion/ddim.py:263]: ms 2.0.0

**Comment by SamitHuang** [comment]
Look nice overall! 
Please also complete the quantitative evaluation using metrics like CLIP-T (same as the implemented CLIP score), CLIP-I, and enrich the comparison results~

**Comment by wtomin** [comment]
> is it possible to add scripts for finding rare token identifiers? can wait until this pr is merged

Sure, finding rare token identifier automatically will be the option for the user in the future. Once this PR is merged, I will work on this.

**Comment by wtomin** [comment]
> Look nice overall! Please also complete the quantitative evaluation using metrics like CLIP-T (same as the implemented CLIP score), CLIP-I, and enrich the comparison results~

The current todo items include the qualitative comparison between naïve finetuning vs.. dreambooth.

As for the quantitative evaluation, the results make more sense when the dreambooth is trained with multiple subjects (e.g., 30 in the dreambooth paper) and we averaged the  CLIP-T and  CLIP-I scores. I'm wondering whether it's worth it, because diffuser  does not provide quantitative results nor qualitive results for dreambooth.

File: examples/stable_diffusion_v2/ldm/models/diffusion/ddim.py (hunk 1 above)
Comment (congw729) [inline examples/stable_diffusion_v2/ldm/models/diffusion/ddim.py:263]: Do we need to define this operator first (maybe in __init__ func) ?

Comment (wtomin) [reply examples/stable_diffusion_v2/ldm/models/diffusion/ddim.py:263]: Not sure if this is a standard requirement, but I can do that.

**Review by congw729 — APPROVED** [review]
LGTM

**Pull request merged**